apigee_edge.module

```diff
@@ -1311,3 +1317,33 @@ function _apigee_edge_user_has_an_app_with_product(string $productName, AccountI
 
   return FALSE;
 }
+
+/**
+ * Indicates that the user synchronization is in progress.
+ *
+ * If the developer-user synchronization is in progress, then saving
+ * developers while saving Drupal user should be avoided.
+ *
+ * @param bool|null $in_progress
+ *   User synchronization state.
+ *
+ * @return bool|null
+ *   TRUE if the user synchronization is in progress, else FALSE or NULL.
+ */
+function _apigee_edge_set_sync_in_progress(?bool $in_progress = NULL) {
+  static $i;
+  if ($in_progress !== NULL) {
+    $i = $in_progress;
+  }
+  return $i;
+}
+
+/**
+ * Gets the user synchronization state.
+ *
+ * @return bool|null
+ *   TRUE if the user synchronization is in progress, else FALSE or NULL.
+ */
+function _apigee_edge_get_sync_in_progress() {
```

Comment:
Probably it should always return a boolean value at least I do not have an example for when we would need to distinguish NULL from FALSE. I also think it should be called as `_apigee_edge_is_sync_in_progress()` because it returns a boolean value.